### PR TITLE
Add missing baselines

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
These baselines were missing, so I added them.

### CODE REVIEW

File: src/Microsoft.Extensions.Hosting/baseline.netcore.json [hunk 1]
```diff
@@ -0,0 +1,397 @@
+{
+  "AssemblyIdentity": "Microsoft.Extensions.Hosting, Version=2.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```
Comment (Tratcher) [inline src/Microsoft.Extensions.Hosting/baseline.netcore.json]: Isn't the baseline supposed to be the prior public release? This component is new, it hasn't shipped before.

Comment (ryanbrandenburg) [reply src/Microsoft.Extensions.Hosting/baseline.netcore.json]: Is that true of the other component too?

Comment (Tratcher) [reply src/Microsoft.Extensions.Hosting/baseline.netcore.json]: No, WindowsServices 2.0 had a net461 version, it should have a baseline. Only the netcoreapp2.0 TFM is new.

Comment (muratg) [reply src/Microsoft.Extensions.Hosting/baseline.netcore.json]: Do we have a way to differentiate the baselines based on the TFM?

Comment (ryanbrandenburg) [reply src/Microsoft.Extensions.Hosting/baseline.netcore.json]: We differentiate Core vs Framework based on the name of the baseline file generated.

**Comment by ryanbrandenburg** [comment]
:up::date:

**Pull request merged**